packages/client/src/views/PrintCertificate/ReviewCertificateAction.tsx

```diff
@@ -351,10 +350,6 @@ function mapStatetoProps(
   const draft = getDraft(declarations, registrationId, eventType)
   const event = getEvent(draft.event)
```

Comment:
@rikukissa @tahmidrahman-dsi I think this might affect you.  You now need to do something like this:

```
{{#ifCond printInAdvance '!==' true}}<image id="image0_414_769" width="1222" height="446" xlink:href="{{registrar.signature}}" />{{/ifCond}}
```